app/src/androidTest/java/mozilla/lockbox/RoutePresenterTest.kt

```diff
@@ -30,11 +29,6 @@ open class RoutePresenterTest {
     @Rule @JvmField
     val activityRule: ActivityTestRule<RootActivity> = ActivityTestRule(RootActivity::class.java)
 
-    @Before
-    fun setUp() {
-        navigator.resetApp(activityRule)
```

Comment:
Why do we no longer have to use `resetApp`?

Reply:
Moving over to the test coordinator means we're now clearing the data in the app before each test: in essence doing a better job of resetting the app that we were doing here.

Result: more stable tests!